inscriptions/ISic001163.xml

```diff
@@ -84,7 +88,7 @@
                                         <depth unit="cm"/>
                                     </dimensions>
                                 </support>
-                                <condition/>
+                                <condition>Fragment</condition>
```

Comment:
you had this right in your earlier version: `<condition ana="#condition.fragment"/>`